concourse/docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:20.04
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-recommends install \
+      python3 python3-pip \
```

Comment:
Should we maybe pin the python version to `3.7` because this is our lowest supported version?

Reply:
We could even use `FROM python:3.7` as a base image.